lib/hammer_cli_katello/sync_plan.rb

```diff
@@ -18,6 +18,7 @@ class InfoCommand < HammerCLIKatello::InfoCommand
 
       output ListCommand.output_definition do
         field :description, _("Description")
+        field :enabled, _("Enabled")
```

Comment:
I would also recommend defining it with Fields::Boolean, so that the user sees 'yes'/'no' versus 'true'/'false'.